src/config.rs

```diff
@@ -1 +1,27 @@
-struct Config {}
+pub enum ProxyMethod {
```

Comment:
What does that mean?
If it's proxy order, I think better name would be `ProxyOrder`, and the fields should be `Sequential`, `Random`

Reply:
So this is the proxychains naming. dynamic chain means for example to use all the station that appear to be working and ignore the nonfunctional ones.